pkg/instaclustr/client.go

```diff
@@ -1553,36 +1553,35 @@ func (c *Client) RestorePgCluster(restoreData *v1beta1.PgRestoreFrom) (string, e
 	}
 
 	var response struct {
-		RestoredCluster string `json:"restoredCluster"`
+		ClusterID string `json:"restoredClusterId"`
 	}
 
 	err = json.Unmarshal(body, &response)
 	if err != nil {
 		return "", err
 	}
 
-	if resp.StatusCode != http.StatusOK {
+	if resp.StatusCode != http.StatusAccepted {
 		return "", fmt.Errorf("status code: %d, message: %s", resp.StatusCode, body)
 	}
 
-	return response.RestoredCluster, nil
+	return response.ClusterID, nil
 }
 
 func (c *Client) RestoreRedisCluster(restoreData *v1beta1.RedisRestoreFrom) (string, error) {
-	url := fmt.Sprintf(APIv1RestoreEndpoint, c.serverHostname, restoreData.ClusterID)
+	url := fmt.Sprintf(APIv2RestoreEndpoint, c.serverHostname, "redis")
```

Comment:
Use models instead of hard-coded values please

Reply:
done